torch/onnx/_internal/exporter.py

```diff
@@ -468,14 +585,70 @@ def export(self) -> ExportOutput:
         )
 
         # Export TorchScript graph to ONNX ModelProto.
-        onnx_model = onnxscript_graph.to_model_proto(self.options.opset_version)
+        onnx_model = onnxscript_graph.to_model_proto(
+            self.options.opset_version,
+            include_initializers=self.options.fake_context is None,
+        )
+
         return torch.onnx.ExportOutput(
             onnx_model,
             self.options.fx_tracer.input_adapter,
             self.options.fx_tracer.output_adapter,
             self.options.diagnostic_context,
         )
 
+    def _assert_fake_tensor_mode(self):
+        has_any_fake_tensor = any(
```

Comment:
you can use `_pytree.tree_any(is_fake_tensor_lambda, self.model_args)`

Same for the rest.